README.md

```diff
@@ -98,63 +120,85 @@ To provide further configuration, add the `custom.kubeapilinter` section to your
 
 [golangci-config-struct]: https://pkg.go.dev/sigs.k8s.io/kube-api-linter/pkg/config#GolangCIConfig
 
-Where fixes are available within a rule, these can be applied automatically with the `--fix` flag.
+Where fixes are available within a rule, these can be applied automatically with the `--fix` flag:
 
 ```shell
-golangci-kube-api-linter run path/to/api/types --fix
+./bin/golangci-lint-kube-api-linter run path/to/api/types --fix
 ```
 
 ### Golangci-lint Plugin
 
 The Kube API Linter can also be used as a plugin for `golangci-lint`.
-To do this, you will need to install the `golangci-lint` binary and then install the Kube API Linter plugin.
+To do this, you will need to install the `golangci-lint` binary and then build the Kube API Linter plugin.
 
 More information about golangci-lint plugins can be found in the [golangci-lint plugin documentation][golangci-lint-plugin-docs].
 
 [golangci-lint-plugin-docs]: https://golangci-lint.run/plugins/go-plugins/
 
+**Important**: The plugin must be built from the vendor directory, not directly from the module path.
```

Comment:
Is this true? Many projects don't use vendor, I expect they can still use the plugin no?